tldw_Server_API/app/main.py

```
@@ -114,6 +115,10 @@
     UnicodeDecodeError,
     ValueError,
 )
+_READINESS_GUARD_EXCEPTIONS = _REQUEST_GUARD_EXCEPTIONS + (
+    ImportError,
+    ModuleNotFoundError,
+)
```

Ruff (0.15.7)

[warning] Consider iterable unpacking instead of concatenation

Replace with iterable unpacking

(RUF005)